content/participants copy/ttjc@sumit121dev/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: dummy student
```

Comment:
Please change the title with your **name** and not **dummy student**

Reply:
@gkanishk sir, shall I raise a new PR or just update the relevant file here only?

Reply:
never mind, just saw the changed label! Thank you so much for working this late. Your commitment inspires us!

Reply:
Your welcome man!!

Reply:
[Noob question] just to be clear, I don't need to participate again at tomorrow morning, my pr will get accepted and I can happily code in peace?